app/facades/plant_facade.rb

```diff
@@ -0,0 +1,7 @@
+class PlantFacade
+  def self.search(search_term)
+    PlantService.search(search_term).map do |plant_data|
+      PlantPoro.new(plant_data)
```

Comment:
Why is this named PlantPoro as opposed to just Plant? Is it because we have a Plant model?